conf.py

```diff
@@ -40,13 +40,36 @@
     ("py:class", "numpy.typing.ArrayLike"),
     # similar for our own type aliases
     ("py:class", "pytket.utils.distribution.T0"),
+    # some packages don't expose all of their classes
+    ("py:class", "qiskit_aer.backends.aerbackend.AerBackend"),
+    ("py:class", "qiskit_aqt_provider.api_client.models_generated.JobResponseRRQueued"),
+    ("py:class", "qiskit_aqt_provider.api_client.models_generated.JobResponseRROngoing"),
+    ("py:class", "qiskit_aqt_provider.api_client.models_generated.JobResponseRRFinished"),
+    ("py:class", "qiskit_aqt_provider.api_client.models_generated.JobResponseRRError"),
+    ("py:class", "qiskit_aqt_provider.api_client.models_generated.JobResponseRRCancelled"),
+    ("py:class", "qiskit_ibm_runtime.models.backend_configuration.QasmBackendConfiguration"),
+    ("py:class", "qiskit_ibm_runtime.models.backend_properties.BackendProperties"),
+    ("py:class", "qujax.utils.CallableArrayAndOptionalArray"),
+    ("py:class", "qujax.utils.CallableOptionalArray"),
     # some other packages it is difficult to link to
     ("py:class", "pathlib._local.Path"),
     ("py:class", "jinja2.nodes.Output"),
+    ("py:class", "numpy.float64"),
+    ("py:class", "qulacs_core.QuantumCircuit"),
+    # matplotlib not always installed and referred to using a string name in pytket-quantinuum
+    ("py:class", "matplotlib.figure.Figure"),
```

Comment:
Would this be solved by always including the matplotlib dependency in the docs build and unstringing the name?

Reply:
Unstringing the name requires matplotlib to always be a defined name in order for the file to even load. Doing this would require us making matplotlib a full dependency rather than just for the calendar extras

Reply:
Okay makes sense, thanks.